app/uk/gov/hmrc/individualsincomeapi/controllers/v1/CommonController.scala

```diff
@@ -58,34 +58,34 @@ abstract class CommonController @Inject() (cc: ControllerComponents) extends Bac
     case e: IllegalArgumentException => ErrorInvalidRequest(e.getMessage).toHttpResponse
   }
 
-  private[controllers] def recoveryWithAudit(correlationId: Option[String], matchId: String, url: String)(implicit
+  private[controllers] def recoveryWithAudit(matchId: String, url: String = "")(implicit
```

Comment:
Just seen why we are dropping `correlationId`. Quite a poor way to do endpoint versioning with the CommonController work. surprised we never revisited this all along. Will be bit of work to get that through, so, we can let this pass by dropping correlationId.

However, can we still keep it and default to None maybe. That way we can revisit this and see how we can align the logic between v1 and v2. Also, chagne `url` to Option[String] and default to None at signature pls.